package.json

```diff
@@ -12,13 +12,12 @@
     "test-node": "./bin/test-node.sh",
     "test-component": "mocha --exit tests/component # --exit for #8839",
     "test-fuzzy": "TYPE=fuzzy npm run test",
-    "test-browser": "npm run build-test && node ./bin/test-browser.js",
+    "test-browser": "node ./bin/test-browser.js",
     "test-memleak": "mocha -gc tests/memleak",
     "eslint": "eslint --cache bin/ packages/node_modules/**/src tests/",
     "dev": "./bin/run-dev.sh",
     "launch-dev-server": "node ./bin/dev-server.js",
     "test": "./bin/run-test.sh",
-    "prepublish": "npm run build",
```

Comment:
I don't understand this change - I assume publish only affects publishing to npm; is there more to it?

Reply:
I've removed it, because it build on `npm install`.

The naming of this script isn't very intuitive, so I looked up it's behaviour and found this article:
https://blog.greenkeeper.io/what-is-npm-s-prepublish-and-why-is-it-so-confusing-a948373e6be1

If the author's intention was to build it for npm, the `publish` script is more suited for this.
I guess: it was more convenient for them to build automatically after install than do it as an extra step...

Reply:
Just looked a bit deeper: `prepublish` is deprecated and npm has introduced two alternative "events": `prepare` and `prepublishOnly`
https://docs.npmjs.com/cli/v10/using-npm/scripts#prepare-and-prepublish

My recent commit adds back the build step for publishing (now via `prepublishOnly`).